corporate/views/remote_billing_page.py

```diff
@@ -118,10 +119,17 @@ def render_tmp_remote_billing_page(
     # This key should be set in both RemoteRealm and legacy server
     # login flows.
     remote_server_uuid = identity_dict["remote_server_uuid"]
-    user_email = identity_dict.get("user_email")
-    user_full_name = identity_dict.get("user_full_name")
+
     remote_realm_uuid = identity_dict.get("remote_realm_uuid")
 
+    user_dict = identity_dict.get("user", {})
+    # Mypy recognizes user_dict as "object" otherwise.
+    # If not empty, this is actually a RemoteBillingUserDict.
+    assert isinstance(user_dict, dict)
```

Comment:
`TypedDict`s are so clunky with mypy :disappointed:

Reply:
It knows `identity_dict` is `Union[RemoteBillingIdentityDict, LegacyServerIdentityDict]`, so it should be able to infer that `identity_dict["user"]` either doesn't exist or is `RemoteBillingUserDict` :confused:

Reply:
We could switch to dataclass serialization I guess.